client-src/default/socket.js

```diff
@@ -32,15 +36,14 @@ const socket = function initSocket(url, handlers) {
         socket(url, handlers);
       }, retryInMs);
     }
-  };
+  });
 
-  sock.onmessage = function onmessage(e) {
-    // This assumes that all data sent via the websocket is JSON.
-    const msg = JSON.parse(e.data);
+  client.onMessage((data) => {
+    const msg = JSON.parse(data);
     if (handlers[msg.type]) {
       handlers[msg.type](msg.data);
     }
-  };
+  });
```

Comment:
Will be great standardize client API, but we can do this in future not in this PR

Reply:
So you want to revert this and have the new client files, but not use them yet?

Reply:
Keep this as is in this PR, we discussion about this in future